spk/octoprint/Makefile

```diff
@@ -1,22 +1,25 @@
-SPK_NAME = octoprint
-SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 1
-SPK_ICON = src/octoprint.png
+SPK_NAME   = octoprint
+SPK_VERS   = $(shell date +%Y%m%d)-devel
```

Comment:
It might be a good idea to wait until the upcoming 1.3.0 release. The latest stable release (1.2.6) doesn't work on DSM due to symlink issues. My PR to OctoPrint with a fix has been merged into the `devel` branch.